var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -311,11 +311,11 @@ class Trilinos(CMakePackage):
     depends_on('swig', when='+python')
 
     patch('umfpack_from_suitesparse.patch', when='@11.14.1:12.8.1')
-    patch('xlf_seacas.patch', when='@12.10.1:%xl')
-    patch('xlf_seacas.patch', when='@12.10.1:%xl_r')
+    patch('xlf_seacas.patch', when='@12.10.1:12.12.1 %xl')
+    patch('xlf_seacas.patch', when='@12.10.1:12.12.1 %xl_r')
+    patch('xlf_seacas.patch', when='@12.12.1 %clang')
```

Comment:
Anyone know why this patch doesn't have the same version range? Since it was previously in the lower section, I have a feeling that it was a mistake.

Reply:
Ah, good catch. I have no idea what these patches are even for. They don't seem necessary for me with clang, because I test nalu against the latest trilinos master with clang nightly using spack to install it and the patch wasn't even being applied.

Reply:
I also test the same thing against the trilinos develop and the patch was being applied and it made difference that I can tell.